tools/bench-asyncio-write.py

```diff
@@ -123,9 +125,9 @@ async def bench(job_title, w, body, base=None):
         base = await bench(t, writes[0], c)
         for w in writes[1:]:
             await bench("", w, c, base)
-    with open("bench.md", "w") as f:
+    async with aiofiles.open("bench.md", "w") as f:
         for line in res:
-            f.write("| {} |\n".format(" | ".join(line)))
+            await f.write("| {} |\n".format(" | ".join(line)))
```

Comment:
But, I don't think we should change this script and add an extra dependency. The blocking call clearly doesn't affect performance as it is the only thing running at this point.

If you wanted to be pedantic, I'd suggest moving it out of the async function instead.

e.g.
```
results = asyncio.run(run_benchmark())
with open(...):
   for l in results:
       ....
```